openpype/hosts/maya/plugins/publish/validate_plugin_path_attributes.py

```diff
@@ -0,0 +1,52 @@
+import os
+
+from maya import cmds
+
+import pyblish.api
+
+from openpype.pipeline.publish import ValidateContentsOrder
+
+
+class ValidatePluginPathAttributes(pyblish.api.InstancePlugin):
+    """
+    Validate plug-in path attributes point to existing file paths.
+    """
+
+    order = ValidateContentsOrder
+    hosts = ['maya']
+    families = ["workfile"]
+    label = "Plug-in Path Attributes"
+
+    def get_invalid(self, instance):
+        invalid = list()
+
+        # get the project setting
+        validate_path = (
+            instance.context.data["project_settings"]["maya"]["publish"]
+        )
+        file_attr = validate_path["ValidatePathForPlugin"]["attribute"]
```

Comment:
This PR has broken maya publishing because THIS `ValidatePathForPlugin` attribute does not exist in settings. It is `ValidatePluginPathAttributes` instead.